app/src/main/java/org/apache/fineract/ui/adapters/PayrollAllocationAdapter.kt

```diff
@@ -28,7 +29,13 @@ class PayrollAllocationAdapter @Inject constructor()
         val (accountNumber, amount, proportional) = payrollAllocation[position]
         holder.tvAccount.text = accountNumber
         holder.tvAmount.text = amount.toString()
-        holder.cbProportional.isChecked = proportional
+        if (proportional) {
+            holder.ivproportional_checked.visibility = View.VISIBLE
+            holder.ivproportional_crossed.visibility= View.GONE
+        }else {
```

Comment:
@varsvat have you locally checked code with `.\gradlew check --scan`. If not I think it will fail **checkstyle** due to no space between else and `}` this brace.

Reply:
Yes, it is failing checkstyle test.

Reply:
What is this , I dont kno about this. Why is it failing ?

Reply:
Run the command that vatsal mentioned in his comment and it will show, btw I can see the issue from the highlighted code you have to put space between "}else"

Reply:
I tried with adding space between } and else , but the check is failing again. Then, I executed the command ./gradlew check --scan before this commit itself , it is failing even then. What to do now ?

Reply:
See in this image , i am on the previous commit by mplodder , but then too , the build is failing .

![image](https://user-images.githubusercontent.com/53621853/112429703-029c2b80-8d63-11eb-81cb-73248d6403d9.png)

Reply:
Try to go through failed test of class you made changes on links that would be available on scrolling up. Read carefully, there would be line number where test is failing. These reports would be less readable but you would figure it out on giving some time to it.

Reply:
Required Changes have been made and now, the TRAVIS build is also successful.